uwblueprint/planet-read
# Implement Return to Translator Flow

## Notion ticket link



[Implement Return to Translator Flow](https://www.notion.so/uwblueprintexecs/Implement-Return-to-Translator-flow-45e3b11cbb3c4ee2aa8a080e0561ab93)



## Implementation description

- Add new button to review page to open confirmation modal 
- In modal, once user presses confirm, call GQL mutation to change the state of the story 
- Once story is in TRANSLATE, reviewer will not be able to change status of lines or click Return to Translator button again 



## Steps to test

1. Go to a review and click Return to Translator, then click confirm when the modal pops up 
2. Story should now be in TRANSLATE stage and the reviewer cannot change status or click button again 
3. Go to the story translation and return the story to REVIEW stage
4. Refresh page and send story back to TRANSLATE 



## What should reviewers focus on?

- Not much, make sure the REVIEW to TRANSLATE flow works on both ends 

## Checklist
